.gitignore

```diff
@@ -6,3 +6,9 @@ __pycache__
 .mypy_cache
 *.bag
 rosdep/
+
+build/
+install/
+log/
+
+.ros/
```

Comment:
These folders should not be made in the src folder if you are running `build` and not `colcon build`